docs/LV_client/tools/mono_matlab_radar_plot.md

```diff
@@ -0,0 +1,27 @@
+# mono_matlab_radar_plot.vi
+
+<p class="img_container">
+<img class="lg_img" src="../mono_matlab_radar_plot.png"/>
+</p>
+
+### Description
+
+Process radar data cube and run a Matlab script to plot Radar data.
```

Comment:
Should we capitalize "Radar Data Cube"? Or perhaps just "Radar"?

Reply:
It is necessary to specify this is the Radar Data Cube